docs/root/intro/arch_overview/load_balancing.rst

```diff
@@ -108,7 +108,9 @@ destination address of the incoming connection was before the connection was red
 Envoy. New destinations are added to the cluster by the load balancer on-demand, and the cluster
 :ref:`periodically <config_cluster_manager_cluster_cleanup_interval_ms>` cleans out unused hosts
 from the cluster. No other :ref:`load balancing type <config_cluster_manager_cluster_lb_type>` can
-be used with original destination clusters.
+be used with original destination clusters. Envoy can pick up the original destination either from 
+the kernel or from a HTTP header called ``x-envoy-original-destination-host``. Please note that
```

Comment:
Make a subsection under this section that you can link to about this header, then, as I asked before, please add an explicit example of what the header should look like. Once you have the subsection that you can link to, you can link to it from elsewhere.